src/hearings/hearings.routes.ts

```diff
@@ -123,6 +124,36 @@ export const ROUTES: Routes = [
           isChildRequired: true
         }
       },
+      {
+        path: 'hearing-completed-summary',
+        resolve: {
+          hearingStageOptions: HearingStageResolver,
+          judicialResponseUsers: JudicialUserSearchResponseResolver,
+          courtLocation: CourtLocationsDataResolver,
+          partyChannels: HearingActualPartyChannelResolverService,
+        },
+        component: HearingCompletedSummaryComponent,
+        canActivate: [HealthCheckGuard],
+        data: {
+          title: 'HMCTS Hearings | Hearing Actuals | Check details',
+          isChildRequired: true
+        }
+      },
+      {
+        path: 'hearing-adjourned-summary',
+        resolve: {
+          hearingStageOptions: HearingStageResolver,
+          judicialResponseUsers: JudicialUserSearchResponseResolver,
+          courtLocation: CourtLocationsDataResolver,
+          partyChannels: HearingActualPartyChannelResolverService,
+        },
+        component: HearingCompletedSummaryComponent,
```

Comment:
so this is using same component HearingCompletedSummaryComponent as hearing-completed-summary? could we extract this hearing-completed-summary for the actual part to a common component? then each these two pages could inject the component with \<exui-hearing-actual-summary\> + the existing \<listing-information-summary\>